docs/environment-setup.md

```diff
@@ -6,6 +6,9 @@
 - [Redis](https://redis.io/) (2 methods)
   - Use [Docker and docker-compose](https://docs.docker.com/install/)
   - Install as a [native application](https://redis.io/topics/quickstart)
+- [Elasticsearch](https://www.elastic.co/what-is/elasticsearch) (2 methods)
+  - Use [Docker and docker-compose](https://docs.docker.com/install/)
+  - Install as a [native application](https://www.elastic.co/guide/en/elastic-stack-get-started/7.6/get-started-elastic-stack.html#install-elasticsearch)
```

Comment:
I'd also put somewhere in here that we also need ES to be running in order for telescope to not crash, I'm not sure if we mention that for Redis as well. @smilegodly was asking why he was getting a protocol error which was probably caused by this.

Reply:
Added a warning in line 13.